frontend/src/ui/Text.tsx

```diff
@@ -6,7 +6,7 @@ const textClasses = cva("text-base font-light text-fg-default", {
   variants: {
     // main variants
     h1: {
-      true: "text-6xl font-bold",
+      true: "text-4xl md:text-6xl font-bold",
```

Comment:
jeśli chcesz responsywne czcionki to pozostałe warianty też by można dostosować

Reply:
Dodalem komentarz do https://github.com/ZachariaszStrus/blog-strapi-next/issues/8